err-urlmatcher/urlmatcher.py

```diff
@@ -0,0 +1,36 @@
+from readability.readability import Document
+import requests
+import re
+
+from errbot import re_botcmd, BotPlugin
+
+# the [^\s\x0f] at the end fixes IRC coloring issue,
+# see https://github.com/mrshu/brutal-plugins/issues/38
+URL_REGEX = '.*((:?https?|ftp)://[^\s/$.?#].[^\s\x0f]*).*'
+TAG_RE = re.compile(r'<[^>]+>')
+WHITESPACE_RE = re.compile(r'\s\s+')
+
+
+class UrlMatcher(BotPlugin):
+    @re_botcmd(pattern=URL_REGEX, prefixed=False)
+    def url_matcher(self, msg, match):
+        url = match.group(0)
```

Comment:
I particularly do not like this line and would love to get the `url` as a parameter to this function (possibly the very last one, after `match`).